docs/book/src/introduction.md

```diff
@@ -1,8 +1,8 @@
-# The Cluster API project<div style="float: right; position: relative; display: inline;"><img src="images/introduction.png" width="160px" /></div>
+# Kubernetes Cluster API<div style="float: right; position: relative; display: inline;"><img src="images/introduction.png" width="160px" /></div>
```

Comment:
I don't agree with the reasoning here. The book describes the current state to me and `Cluster API project` reflects that state to me.

Reply:
should definitely have Kubernetes in the title, regardless though?

Reply:
SIG Docs would certainly say it should.

Reply:
I am not blocking the inclusion of `Kubernetes` in the title but advocate for the inclusion of `project`.

I don't mind too much at the end of the day, just wanted to share my viewpoint :)